scss/tech.scss

```diff
@@ -52,13 +39,30 @@
 }
 
 .container {
-  margin: 0 auto;  
   flex-direction: column; 
+  margin: 0 auto;  
   max-width: 1050px;
   padding-top: 80px;
 }
 
 .display-wrap {
   display: flex;
-  flex-wrap: wrap;
+  flex-wrap: wrap;  
+}
+
+.grid {
+  align-items: center;
+  display: grid;
+  grid-template-columns: repeat(3, max-content);
+  justify-content: center;
+  
+  :first-child {
+    grid-row: 1/3;
+  } 
+}
+
+.border {
+  border-right: 2px solid $earls-green;
+  height: 535px;
```

Comment:
I think there's no need for this, the height should be automatically set :thinking: